consensus/builddev.go

```diff
@@ -9,15 +9,15 @@ import (
 const (
 	DEBUG = true
 
-	BlockSizeLimit        = 1024 * 1024            // Blocks cannot be more than 1MB.
-	BlockFrequency        = Timestamp(10)          // In seconds.
-	TargetWindow          = BlockHeight(80)        // Number of blocks to use when calculating the target.
-	MedianTimestampWindow = 11                     // Number of blocks that get considered when determining if a timestamp is valid. Should be an odd number.
-	FutureThreshold       = Timestamp(3 * 60 * 60) // Seconds into the future block timestamps are valid.
-	SiafundCount          = Currency(10 * 1000)    // The total (static) number of siafunds.
-
-	InitialCoinbase = Currency(300 * 1000)
-	MinimumCoinbase = Currency(30 * 1000)
+	BlockSizeLimit        = 1024 * 1024 // Blocks cannot be more than 1MB.
+	BlockFrequency        = 10          // In seconds.
+	TargetWindow          = 80          // Number of blocks to use when calculating the target.
+	MedianTimestampWindow = 11          // Number of blocks that get considered when determining if a timestamp is valid. Should be an odd number.
+	FutureThreshold       = 3 * 60 * 60 // Seconds into the future block timestamps are valid.
+	SiafundCount          = 1e4         // The total (static) number of siafunds.
+
+	InitialCoinbase = 3e5
+	MinimumCoinbase = 3e4
```

Comment:
can we have everything be e3/6/9/etc?

10e3
300e3
30e3

While we're at it, BlockSizeLimit can just be
1e6